developer-docs-site/docs/guides/guide-for-system-integrators.md

```diff
@@ -25,16 +25,29 @@ This guide will overview the following topics for integrating with Aptos:
 
 ## Getting Started
 
-* Obtain the [Aptos CLI](../cli-tools/aptos-cli-tool/install-aptos-cli/) -- recommend by source and targeting the `main` branch as the codebase will be under many changes for the next couple of weeks.
-* Start a [local node with a Faucet](../cli-tools/aptos-cli-tool/use-aptos-cli#running-a-local-testnet): `aptos node run-local-testnet --with-faucet`.
-* Export the appropriate `FAUCET_URL`: `export FAUCET_URL=http://127.0.0.1:8081"`.
-* Export the appropriate `NODE_URL`: `export NODE_URL=http://127.0.0.1:8080"`.
+To get started, you will need:
+
+1. The Aptos CLI tool.
+2. A local testnet with a validator node that has the faucet service turned on, and
+3. The correct version of the Aptos SDK. The version is important to ensure that the SDK code builds the clients that talk to the local tesnet. 
+
+Follow the below steps to set up the above three:
+
+1. Follow the instructions for Aptos CLI in [Install with cargo](/cli-tools/aptos-cli-tool/install-aptos-cli#install-with-cargo) and install the CLI tool. Make sure you that when you execute Step 2, you use `--branch main` to target the `main` branch, and not the `--branch devnet` as the codebase is undergoing many changes during the next 2-3 weeks.
```

Comment:
might be good to express why one might use one versus the other:

main branch is for local development -- e.g., local node with faucet
devnet branch is for development on dvent